src/measure_theory/prod_group.lean

```diff
@@ -0,0 +1,205 @@
+/-
+Copyright (c) 2021 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import measure_theory.prod
+import measure_theory.group
+
+/-!
+# Measure theory in the product of groups
+
+In this file we show properties about measure theory in products of topological groups
+and properties of iterated integrals in topological groups.
+
+These lemmas show the uniqueness of left invariant measures on locally compact groups, up to
+scaling. In this file we follow the proof and refer to the book *Measure Theory* by Paul Halmos.
+
+The idea of the proof is to use the translation invariance of measures to prove `μ(F) = c * μ(E)`
+for two sets `E` and `F`, where `c` is a constant that does not depend on `μ`. Let `e` and `f` be
+the characteristic functions of `E` and `F`.
+By translation invariance the map `(x, y) ↦ (y * x, x⁻¹)` is measure-preserving, which means that
+```
+  ∫ x, ∫ y, h x y ∂ν ∂μ = ∫ x, ∫ y, h (y * x) x⁻¹ ∂ν ∂μ
```

Comment:
```suggestion
  ∫ x, (∫ y, h x y ∂ν) ∂μ = ∫ x, (∫ y, h (y * x) x⁻¹ ∂ν) ∂μ
```
I was confused at first because often one writes `\int_x \int y f dm(x) dm(y)`, which is *not* what the expression above means. Adding parentheses helps.

Reply:
Sorry, I'm not sure what you're saying here. What is your alternative interpretation of the double integral? The expression on the left-hand side can also be written as `∫ ∫ h ∂ν ∂μ` or `∫_x ∫_y h ∂ν(y) ∂μ(x)` (which all mean the same to me).